common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala

```diff
@@ -105,7 +107,7 @@ object Mappings {
 
   val collectionMapping = withIndexName("collection", nonDynamicObj(
     "path" -> nonAnalysedList("collectionPath"),
-    "pathId" -> nonAnalyzedString,
+    "pathId" -> (nonAnalyzedString ++ copyTo("collections.pathHierarchy")),
```

Comment:
[Done](https://github.com/guardian/grid/commit/9763f16dfb62125339dcc443f5687fb4a08a9145). Interestingly talking to @akash1810 and @NickPapacostas about using it for a mega free text search field.